.gitignore

```diff
@@ -274,3 +274,5 @@ ready-for-test.sh
 reset-qa.sh
 reset-local.sh
 ready-for-integration.sh
+e2e/env/*.postman_environment.json
```

Comment:
I don't think this .gitignore line works. I have a file in here that git shows as a new file but that I think needs to be ignored

Reply:
Pushed a fix